csharp/Platform.Unsafe/MemoryBlock.cs

```diff
@@ -8,32 +8,19 @@
 
 namespace Platform.Unsafe
 {
-    /// <summary>
-    /// <para>
-    /// Represents the memory block.
-    /// </para>
-    /// <para></para>
+    /// <summary><para>Creating a custom memory block via <see cref="Parallel"/>.</para><para>Создание пользовательского блока памяти через <see cref="Parallel"/>.</para>
     /// </summary>
     public static unsafe class MemoryBlock
     {
         /// <summary>
-        /// <para>
-        /// Zeroes the pointer.
-        /// </para>
-        /// <para></para>
+        /// <para>Zeroes the <paramref name="pointer"/>.</para><para>Обнуляет <paramref name="pointer"/>.</para>
```

Comment:
Not the pointer is zeroed, but the data that pointer points to. It is important to note that only the number of bytes specified in `capacity` are set to zero.